src/languageHandler.tsx

```diff
@@ -188,22 +189,35 @@ export function substitute(text: string, variables: IVariables, tags: Tags): str
 export function substitute(text: string, variables?: IVariables, tags?: Tags): string | React.ReactNode {
     let result: React.ReactNode | string = text;
 
-    if (variables !== undefined) {
-        const regexpMapping: IVariables = {};
+    const hasVariables = !isUndefined(variables);
+    const hasTags = !isUndefined(tags);
```

Comment:
Seems a bit obtuse to pull in a library to perform `x === undefined`
https://github.com/lodash/lodash/blob/4.17.15/lodash.js#L12198

Reply:
@t3chguy we have lodash anyways, so why not use it?

Reply:
in theory that bit of lodash might be tree-shaken out if its not used elsewhere. Seems overkill to call a method for a simple comparison, also IMO it is less readable once you add the negation.

`x !== undefined` is more readable for me personally than `!isUndefined(x)` - opinion of course

Reply:
we need a tree-shaking plugin for lodash to even be able to tree shake it, hoping it's not being globally updated. I guess that's a thing we could have a look at separately.
I'll do your suggestion then.